src/test/java/ru/yandex/qatools/embed/postgresql/TestMultipleInstance.java

```diff
@@ -0,0 +1,69 @@
+package ru.yandex.qatools.embed.postgresql;
+
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import ru.yandex.qatools.embed.postgresql.distribution.Version;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.Statement;
+
+public class TestMultipleInstance {
+	@Test
+	public void testRunTwoConsistentlyInstanceDifferenceVersions() throws Exception {
```

Comment:
"test" prefix in the method name is actually redundant. Let's call them in accordance with the check, which they perform (e.g. "itShouldAllowToRunTwoInstancesWithDifferentVersions")